src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -73,9 +75,27 @@ export default function NotificationsPane({
                 >
                   <div style={{ display: 'flex' }}>
                     <Avatar src={shane} variant="circular" />
-                    <Text style={{ maxWidth: 200, margin: '0 20px' }}>
-                      {`${senderName} sent you a collaboration request`}
-                    </Text>
+                    {senderName !== 'N/A' && (
+                      <Text
+                        style={{ maxWidth: 200, margin: '0 20px' }}
+                      >
+                        {`${senderName}`}
```

Comment:
is there a space between senderName and "sent you a collaboration request"? Just making sure. There is also a way to inject sendername directly into the intl message as a value.

Reply:
Added a space. Done.